soccer/src/soccer/strategy/agent/position/robot_factory_position.cpp

```diff
@@ -19,7 +19,7 @@ RobotFactoryPosition::RobotFactoryPosition(int r_id) : Position(r_id, "RobotFact
     std::string node_name{"robot_factory_position_"};
     _node = std::make_shared<rclcpp::Node>(node_name.append(std::to_string(robot_id_)));
     override_play_sub_ = _node->create_subscription<rj_msgs::msg::OverridePosition>(
-        "override_position_for_robot", 1,
+        "override_position_for_robot_" + std::to_string(robot_id_), 1,
```

Comment:
```suggestion
        "override_position/robot_" + std::to_string(robot_id_), 1,
```
I recommend doing this so in ros tools we get a hierarchy

Reply:
ideally we also put this in some constants header but i know we're not great at that